src/oc_erchef/apps/chef_db/priv/pgsql_statements.config

```diff
@@ -20,7 +20,7 @@
 {ping, <<"SELECT 'pong' as ping LIMIT 1">>}.
 
 %% Query to count the number of nodes for license checks
-{count_nodes, <<"SELECT COUNT(*) as count FROM nodes">>}.
+{count_nodes, <<"SELECT COUNT(*) as count FROM nodes as n LEFT JOIN orgs as o ON n.org_id = o.id WHERE o.id IS NOT NULL;">>}.
```

Comment:
Might be clearer to `SELECT COUNT(*) as count FROM nodes, orgs WHERE orgs.id = nodes.org_id`?